core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -41,6 +42,22 @@ oppia.factory('SimpleEditorManagerService', [
       }
     };
 
+    var DEFAULT_ITEM_INTERACTION = {
```

Comment:
Though it might now make sense to have a dict: ALLOWED_QUESTION_TYPES, formatted like:
```
  {
    multipleChoice: {
      interactionId: ...
      defaultCustomizationArgs: ...
    },
    ...
  }
```